lib/rust/ensogl/core/src/display/style/sheet.rs

```diff
@@ -240,10 +242,17 @@ where T: Into<Data>
     }
 }
 
-impl TryFrom<String> for Value {
-    type Error = <Data as TryFrom<String>>::Error;
-    fn try_from(s: String) -> Result<Self, Self::Error> {
-        s.try_into().map(Self::Data)
+impl FromStr for Value {
+    type Err = <Data as FromStr>::Err;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        s.parse().map(Self::Data)
+    }
+}
+
+impl<Err> TryFrom<Result<Value, Err>> for Value {
+    type Error = Err;
+    fn try_from(t: Result<Value, Err>) -> Result<Value, Err> {
+        t
```

Comment:
This is strange. What is it for?

Reply:
As we discussed, I removed this and added a `set_parsed` function in the `Theme` type instead.